ql/src/test/org/apache/hadoop/hive/ql/lockmgr/TestDbTxnManager2.java

```diff
@@ -1565,7 +1565,7 @@ public void testWriteSetTracking10() throws Exception {
     driver.compileAndRespond("update tab1 set b = 7 where b=2", true);
     txnMgr2.acquireLocks(driver.getPlan(), ctx, "T2");
     List<ShowLocksResponseElement> locks = getLocks(txnMgr2);
-    Assert.assertEquals("Unexpected lock count", 2, locks.size());
+    Assert.assertEquals("Unexpected lock count", 3, locks.size());
```

Comment:
Why is this change?

Reply:
Update branch was split to an insert and a delete to the same table. The number of locks increased because a side effect of the way Hive acquires locks when executing multi insert statements.